src/main/java/tutorlink/commandpackage/AddAssignmentCommand.java

```diff
@@ -1,32 +1,30 @@
 package tutorlink.commandpackage;
 
 import tutorlink.assignmentpackage.Assignment;
-import tutorlink.coursepackage.Course;
 import tutorlink.exceptionspackage.ItemNotFoundException;
 import tutorlink.resultpackage.CommandResult;
 
-public class AddAssignmentCommand extends Command{
+public class AddAssignmentCommand extends Command {
     public static final String COMMAND_WORD = "add_assignment";
     public static final String FORMAT_ERROR_MESSAGE = "Error, expected format: "
-            + COMMAND_WORD + " i/MATRIC NUMBER " + "c/COURSE_ID " + "n/ASSIGNMENT_DESCRIPTION "
+            + COMMAND_WORD + " i/MATRIC NUMBER " + "n/ASSIGNMENT_DESCRIPTION "
             + "s/RECEIVED_SCORE " + "t/TOTAL_SCORE " + "w/WEIGHTING";
-    public static final String REGEX = "add_assignment\\s+i/(A\\d{7}[A-Z])\\s+c/([A-Z]{2,3}\\d{4})\\s+n/([\\s\\S]+?)" +
+    public static final String REGEX = "add_assignment\\s+i/(A\\d{7}[A-Z])\\s+n/([\\s\\S]+?)" +
             "\\s+s/(\\d+(\\.\\d+)?)\\s+t/(\\d+(\\.\\d+)?)\\s+w/(\\d+(\\.\\d+)?)";
     public static final String INVALID_WEIGHTING_MESSAGE = "Error! Weighting must be 0 < w < 1";
 
     private static final String SUCCESSFUL_MESSAGE = "Assignment (Description: %s, Score: %s/%s) added successfully "
-            + "to Course %s for student (Matric Number: %s)";
+            + "for student (Matric Number: %s)";
+
     private String matricNumber;
```

Comment:
As per our latest UML diagram, an "Assignment" (we should rename this Class to Grade) object does not have a reference to a Student object since the association between Student and Grade is via an Association object instead.